File: TransCoder_tokenized_test_set_functions/java/LONGEST_SUBSEQUENCE_WHERE_EVERY_CHARACTER_APPEARS_AT_LEAST_K_TIMES.java
static void longestSubseqWithK ( String str, int k ) {
  int n = str . length ( );
  int freq [ ] = new int [ MAX_CHARS ];
  for ( int i = 0;
  i < n;
  i ++ ) {
    freq [ str . charAt ( i ) - 'a' ] ++;
  }
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( freq [ str . charAt ( i ) - 'a' ] >= k ) {
      System . out . print ( str . charAt ( i ) );
    }
  }
}

File: TransCoder_tokenized_test_set_functions/cpp/LONGEST_SUBSEQUENCE_WHERE_EVERY_CHARACTER_APPEARS_AT_LEAST_K_TIMES.cpp
void longestSubseqWithK ( string str, int k ) {
  int n = str . size ( );
  int freq [ MAX_CHARS ] = {
    0 };
    for ( int i = 0;
    i < n;
    i ++ ) freq [ str [ i ] - 'a' ] ++;
    for ( int i = 0;
    i < n;
    i ++ ) if ( freq [ str [ i ] - 'a' ] >= k ) cout << str [ i ];
  }